subprojects/common/analysis/src/test/java/hu/bme/mit/theta/analysis/expl/ExplStmtTransFuncTest.java

```diff
@@ -98,7 +82,7 @@ public void testBottom() {
     public void testComplex1() {
         final ExplStmtTransFunc transFunc = ExplStmtTransFunc.create(solver, 0);
         final ExplState sourceState = ExplState.top();
-        final ExplPrec prec = ExplPrec.of(ImmutableSet.of(x, y));
+        final ExplPrec prec = ExplPrec.of(Set.of(x, y));
```

Comment:
Why was this changed?

Reply:
It was a sonar detected issue. I fixed some of them to improve overall code quality